clients/vmdisplay/vmdisplay-server-hyperdmabuf.cpp

```diff
@@ -117,28 +124,26 @@ int HyperDMABUFCommunicator::recv_metadata(void **buffer)
 {
 	int len;
 
-	struct vm_header last_hdr;
-	int num_buffers = 0;
 	struct hyper_dmabuf_event_hdr *event_hdr;
 
 	while (1) {
 		/*
 		 * In case when we received alreay buffer for next frame,
 		 * append its metadata to new frame metadata.
 		 */
-		if (hdr)
-			memcpy(&last_hdr, hdr, sizeof(*hdr));
 
-		if (hdr && hdr->counter != last_counter) {
+		if (hdr && hdr->counter != last_counter[hdr->output]) {
 			/* Copy metatadat to mmaped file of given output */
 			memcpy((char *)buffer[hdr->output] +
-			       offset[hdr->output], buf_info,
-			       sizeof(struct vm_buffer_info));
-			offset[hdr->output] += sizeof(struct vm_buffer_info);
-			num_buffers++;
-			last_counter = hdr->counter;
+					offset[hdr->output], buf_info,
```

Comment:
white spaces?

Reply:
It's tab spaces but looks indent is not matching existing coding style. Is there a wiki regarding to coding style?

Reply:
I believe it follows style for open source weston.
https://gitlab.freedesktop.org/wayland/weston/blob/master/CONTRIBUTING.md
@satyeshwar: Please correct me if I am wrong